services/movie_session.py

```diff
@@ -42,3 +42,14 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
```

Comment:
The 'delete_movie_session_by_id' function should handle the case where a movie session with the given ID does not exist. Consider using 'get_object_or_404' or handling the 'DoesNotExist' exception to provide a more user-friendly error message.